### DISCUSSION TITLE
ephys-link version for pinpoint

### DISCUSSION THREAD

**Discussion by rebz444** [VirtualBrainLab/ephys-link#264] [Q&A] [answered]
I currently have ephys-link v0.9.14, and sensapex uMP4 manipulators. I have ephys-link running and am trying to use pinpoint to control these manipulators, but pinpoint says "Ephys Link is outdated. please update to >= v1.0.0. However, it seems like v0.9.14 is the newest version from this repo? Where can I get a version that is >= v1.0.0?
<img width="1438" alt="ephys-link" src="https://github.com/VirtualBrainLab/ephys-link/assets/63360362/af7c71fa-d165-4976-bbff-7d7af058b97a">

**Comment by kjy5** [comment, marked as answer]
Hi, apologies for the late reply. How are you currently installing and managing Pinpoint? Through the new year, we have been re-organizing the way people can access Pinpoint and Ephys Link.

The current best way to get the latest version of Pinpoint (v1.1) is via Steam. Ephys Link has been updated on PyPI to v1.1.1 and we also publish an executable on the releases page (all dependencies needed are packaged in the exe).

Please let me know if you are still experiencing these issues after updating to the latest versions!
